frontend/src/hooks/useLoading.ts

```diff
@@ -2,11 +2,13 @@ import { createRef, useLayoutEffect } from 'react';
 
 type refElement = HTMLDivElement;
 
-const useLoading = (loading: boolean) => {
+const useLoading = (loading: boolean, label = 'true') => {
```

Comment:
IMO this API is too specific. As a user of useLoading I'd expect (loading: boolean, selector = '[data-loading]') that I can override. label that goes into data-loading selector is not very intuitive from the user's perspective.